pkg/shell/common.go

```diff
@@ -95,11 +96,11 @@ func CheckWritableDir(path string) error {
 // skip making the proposed changes and continue execution (in deploy command)
 // only if the user responds with "y" or "yes" (case-insensitive)
 func ApplyChangesChoice(c ProposedChanges) bool {
-	log.Printf("Summary of proposed changes: %s", strings.TrimSpace(c.Summary))
+	logging.Info("Summary of proposed changes: %s", strings.TrimSpace(c.Summary))
 	var userResponse string
 
 	for {
-		fmt.Print(`(D)isplay full proposed changes,
+		logging.Info(`(D)isplay full proposed changes,
```

Comment:
I think this one should stay `fmt.Printf` since it's user prompt it should always go to stdout.

Reply:
I changed to back to `fmt.Print` since I assumed that's what you meant?